Makefile

```diff
@@ -1,44 +1,14 @@
-TESTS = `find test/*.test.js`
-REPORTER = spec
-TIMEOUT = 5000
-MOCHA_OPTS =
-
 install:
-	npm install
+	npm i
 
 test:
-	NODE_ENV=test ./node_modules/mocha/bin/mocha \
-		--harmony\
-		--reporter $(REPORTER) \
-		--timeout $(TIMEOUT) \
-		--require should \
-		$(MOCHA_OPTS) \
-		$(TESTS)
-
-
-test-cov:
-	NODE_ENV=test node --harmony \
-		node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha \
-		-- -u exports \
-		--reporter $(REPORTER) \
-		--timeout $(TIMEOUT) \
-		--require should \
-		$(MOCHA_OPTS) \
-		$(TESTS)
+	NODE_ENV=test npm test
 
-test-travis:
-	NODE_ENV=test node --harmony \
-		node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha \
-		--report lcovonly \
-		-- -u exports \
-		--reporter $(REPORTER) \
-		--timeout $(TIMEOUT) \
-		--require should \
-		$(MOCHA_OPTS) \
-		$(TESTS)
+test-only:
+	NODE_ENV=test npm run-script test-only
 
 autod:
-	./node_modules/.bin/autod -w -e example --prefix=~ --keep=supertest,debug, --semver=koa@1
+	npm i autod && autod -w -e example --prefix=~ --keep=debug, --semver=koa@1
```

Comment:
Fixed in https://github.com/DaAwesomeP/koa-redis/commit/94e0b15344d4ea45b0e037ee9eafa84497721049